src/coreclr/vm/prestub.cpp

```diff
@@ -2750,10 +2750,16 @@ extern "C" void STDCALL ExecuteInterpretedMethod(TransitionBlock* pTransitionBlo
 
     InterpMethodContextFrame interpFrame = {0};
     interpFrame.startIp = (int32_t*)byteCodeAddr;
+    interpFrame.ip = interpFrame.startIp;
     interpFrame.pStack = sp;
     interpFrame.pRetVal = sp;
 
+    InterpreterEntryFrame interpreterEntryFrame(pTransitionBlock, &interpFrame);
+
+    // Interpreter-TODO: catch exceptions stemming from the InterpExecMethod and call the managed exception handling
```

Comment:
Not sure what you mean here by catching exceptions thrown by `InterpExecMethod`. No C++ exceptions should ever be thrown at this point.

Reply:
Not that method itself, but its callees can throw.

Reply:
But `InterpExecMethod` should catch all C++ exceptions and convert them to managed exceptions if necessary, right ? If we reach here, we got rid of all managed frames and it would be incorrect. Just to clarify.